### ISSUE TITLE
The AIML40 upload script gives an error

### ISSUE THREAD

**Issue by jimbobbennett** [microsoft/ignite-learning-paths-training-aiml#59]
Running the command to upload the dataset:

`python upload_dataset.py -s [subscription_id] -w absa_space -g absa -f clothing_automl.xlsx`

This gives the following error:

```sh
Traceback (most recent call last):
  File "upload_dataset.py", line 4, in <module>
    from azureml.core import Workspace,Dataset
ModuleNotFoundError: No module named 'azureml'
```

**Comment by jimbobbennett** [comment]
Installing `azureml.core` via pip allows the code to run, but gives a different error:

```sh
Traceback (most recent call last):
  File "upload_dataset.py", line 18, in <module>
    ws = Workspace(args.s, args.g, args.w)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/core/workspace.py", line 185, in __init__
    auto_rest_workspace = _commands.get_workspace(auth, subscription_id, resource_group, workspace_name)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/_project/_commands.py", line 363, in get_workspace
    resource_error_handling(response_exception, WORKSPACE)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/_base_sdk_common/common.py", line 338, in resource_error_handling
    raise ProjectSystemException("{resource_type} not found.".format(resource_type=resource_type))
azureml.exceptions._azureml_exception.ProjectSystemException: ProjectSystemException:
	Message: Workspace not found.
	InnerException None
	ErrorResponse 
{
    "error": {
        "message": "Workspace not found."
    }
}
```

The deploy to azure button was used to create all the resources.

**Comment by jimbobbennett** [comment]
More info - this assumes the Resource Group is named `absa`, which isn't given as a requirement anywhere, apart from a small note about using absa as the name when using the CLI.

If I change this to my resource group name I get an error about not being able to find the data file:

```sh
	Message: '/Users/jimbennett/GitHub/ignite-learning-paths-training-aiml/aiml40/clothing_automl.xlsx' does not point to a file. Please upload the file to cloud first if running in a cloud notebook.
	InnerException None
	ErrorResponse 
{
    "error": {
        "code": "UserError",
        "message": "'/Users/jimbennett/GitHub/ignite-learning-paths-training-aiml/aiml40/clothing_automl.xlsx' does not point to a file. Please upload the file to cloud first if running in a cloud notebook."
    }
}
```

The command assumes the upload script and data file are in the same location. The command should be updated to reference the data file in the correct folder

**Comment by jimbobbennett** [comment]
Updating the path to be correct leads to further errors:

```sh
ModuleNotFoundError: No module named 'azureml.dataprep'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "upload_dataset.py", line 28, in <module>
    dset = Dataset.Tabular.from_delimited_files(path=datastore_paths)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/data/_loggerfactory.py", line 78, in wrapper
    return func(*args, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/data/dataset_factory.py", line 219, in from_delimited_files
    dataflow = dataprep().read_csv(_normalize_path(path),
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/azureml/data/_dataprep_helper.py", line 42, in dataprep
    raise ImportError(_dataprep_missing_error)
ImportError: Could not import package "azureml-dataprep". Please ensure it is installed by running: pip install "azureml-dataprep[fuse,pandas]"
```

The fix is to pip install `azureml-dataprep`

**Comment by jimbobbennett** [comment]
Summary:

* The instructions need to include pip installing the required packages
* The resource group name should be referred to and where it is used without explanation, the usage should be explained
* The upload command should refer to the data file in its sub folder.=

**Comment by shwars** [comment]
Thank you for raising this problem, @jimbobbennett!
In fact it turns out that there is no easy way to upload excel dataset programmatically, therefore we have decided to remove the option to upload dataset and create clusters through Python scripts, replacing it with manual instructions on how to do it. It would also remove the burden of having to install local Python environment altogether.
Updated instructions would be available shortly.
